src/daemon/daemon.cpp

```diff
@@ -1680,6 +1680,26 @@ try // clang-format on
             delayed_shutdown_instances.erase(name);
         }
 
+        try
+        {
+            auto& sshfs_mounts = mount_threads.at(name);
+            for (const auto& sshfs_mount : sshfs_mounts)
+            {
+                mpl::log(mpl::Level::debug, "delete",
+                         fmt::format("Stopping mount '{}' in instance \"{}\"", sshfs_mount.first, name));
+                sshfs_mount.second->stop();
+            }
+        }
+        catch (const std::out_of_range&)
+        {
+            mpl::log(mpl::Level::debug, "delete", fmt::format("No mounts to stop for instance \"{}\"", name));
```

Comment:
If I am not mistaken, all other logs in `daemon.cpp`  use the constant `category` as the 2nd param. Wouldn't it be better to stick to that here?

Reply:
Yeah, I thought about that.  I guess I should use that instead for consistency's sake, but I do think at a later time, we should change that to be tell operation in the daemon logged the message.

For example, category would just be `daemon` and we don't really have a clue in the logs what operation logged that message.

Reply:
I see your point. I don't have a strong opinion on category contents yet (i.e. entity vs operation). It is a question of how you split information, I suppose. The message can be constructed in such a way that it conveys all necessary meaning; or you can leave a bit of info to the category. 

But quickly screening relevant messages based on the category works better if it is used consistently. Of course if you see "daemon" everywhere that doesn't help much, but a single "delete" could be confusing ("didn't that originate on the daemon like the other msgs?"). So perhaps a separate PR for that?

Reply:
Yep, separate PR.  I'll change this PR to use `category` for now.

Reply:
I just noticed the category is not part of the default output of `journalctl`. Wouldn't it be better to clarify in the message? Something like "No mounts to stop for instance ... before deleting"

Reply:
Hmm, I never noticed that about journalctl.  I imagine there are lots of places that would need better wording in this case.  Let's leave that one alone for now as well:)

Reply:
BTW, I changed the category string.